src/utils/uint256.ts

```diff
@@ -10,14 +10,34 @@ export { UINT_128_MAX, UINT_256_MAX };
 /**
  * Convert Uint256 to bigint
  * Legacy support Export
+ * @param uint256 Uint256 value to convert to bigint
+ * @returns {bigint} BigInt representation of the input Uint25
+ * @example
+ * ```typescript
+ * const uint256Value: Uint256 = { low: 1234567890, high: 1 };
+ * const result = uint256ToBN(uint256Value);
+ * // result = 519229685234822890n
```

Comment:
I have executed your example, and I don't have the same result : 
`340282366920938463463374607433002779346n`
You have to be more attentive ...